store/storetest/post_store.go

```diff
@@ -2067,8 +2067,9 @@ func testPostStoreGetOldest(t *testing.T, ss store.Store) {
 	o2.CreateAt = 1
 	o2 = (<-ss.Post().Save(o2)).Data.(*model.Post)
 
-	r1 := (<-ss.Post().GetOldest()).Data.(*model.Post)
+	r1, err := ss.Post().GetOldest()
 
+	require.NoError(t, err)
```

Comment:
The error is failing because it should be `require.Nil`, because we are not returning an `error` we are returning a `model.AppError` which match the `error` interface. is not exactly the same. using here, `require.Nil` should solve the test failure.

Reply:
Thanks for the advice :+1: . I set it to `require.Nil` and `post_store.go` test is passing locally, but it cannot be built on Jenkins. I don't know why. Logs aren't very descriptive :/

Reply:
Probably because I need to upgrade the enterprise repo PR